pyquil/quil.py

```diff
@@ -832,7 +832,7 @@ def out(self, *, calibrations: Optional[bool] = True) -> str:
         if calibrations:
             return self._program.to_quil()
         else:
-            return self._program.into_simplified().to_quil()
+            return self.remove_quil_t_instructions().out()
```

Comment:
This does more than remove calibrations, no? This also removes DELAY, PULSE, etc, which do belong in the program

Reply:
Right, good catch, used a different predicate to remove just the calibration instructions.